melos.yaml

```diff
@@ -51,6 +51,19 @@ scripts:
     run: melos exec flutter pub run dartdoc
     description: Run dartdoc checks for all packages.
 
+  doc-setup:
+    run: |
+      echo '\033[1;32mChecking python version:\033[m' &&
+      python --version &&
+      echo '\n\033[1;32mInstalling required python modules:\033[m' &&
+      (python -c 'import sys; sys.exit(0 if sys.version_info>=(3,8) else 2)' ||
+       (echo '\033[1;31mError: Python 3.8+ is required\033[m' && false)) &&
+      python -m pip install -r "$MELOS_ROOT_PATH/doc/_sphinx/requirements.txt" &&
+      echo '\n\033[1;32mInstalling dartdoc_json:\033[m' &&
+      dart pub global activate dartdoc_json &&
```

Comment:
We should be able to run this without `global` I think, and we should probably also check whether `dart` exists on the path and if not use `flutter` instead.

Reply:
`dart pub activate dartdoc_json` wouldn't work -- or what did you have in mind?

Reply:
Not sure what I had in mind, I've never done that.
But feels like there must be a way to not install it globally on the system.
Maybe it would actually work to download it as a dev_dependency as you suggested before and then point to where it is downloaded. But also maybe it is not worth it...
Maybe just keep it like it is. :smile: